File: gabriel/Problema0192/Problema0192.py

n0 = (int(input("Digite o primeiro valor de um intervalo fechado: ")))
nx = (int(input("Digite o último valor de um intervalo fechado: ")))

if(nx < n0):       
    for i in range(nx, n0+1):
        print(str(i))

else: 
    for i in range(n0, nx+1):
        print(str(i))
    

File: gabriel/Problema0192/Problema0192.js

    var n0 = (parseInt(prompt("Digite o primeiro valor de um intervalo fechado: ")));
    var nx = (parseInt(prompt("Digite o último valor de um intervalo fechado: ")));
    
    if(n0 < nx){
        for(var i = n0; i<nx + 1; i++){
            console.log(i);
        }
    }

    else{
        for(var i=nx; i<n0+1; i++){
            console.log(i);
        }
    }
 